kernel/fcbfns.c

```diff
@@ -502,13 +502,32 @@ UBYTE FcbDelete(xfcb FAR * lpXfcb)
   return result;
 }
 
+static void FcbExpandAsterisks(char *p, int len)
+{
+  int i, afound;
+
+  for (i = 0, afound = 0; i < len; i++) {
+    if (p[i] == '*' || afound) {
```

Comment:
But does this properly handle, say, `a*.b*`?
Or `a*.bat*`?
I mean, you don't seem to be handling dot, which
may have a separate wildcard.
Also not sure if the cases like `*a.*t` are valid?
I think the code should be `if (p[i] == '*' || (afound && p[i] == ' ')) {`
or something like that.

Reply:
The page I linked to in the commit explains better, but essentially it's very simple and not rich like unix wildcards.
so
`*a.b*` ==> `???????.b??`
`a*.bat*` is illegal
`*a.*t` -> `????????.???`

Reply:
There's no dot as the input is an FCB name (8+3 chars, no dot, no \\0).